src/conductor/checks.py

```diff
@@ -306,10 +306,10 @@ class GSheetChecker():
     client = None
     worksheet = None
 
-    def __init__(self, table, sheet_id, worksheet_name, testing=False):
+    def __init__(self, table, sheet_id, worksheet_name, sa):
```

Comment:
**nitpick**: More descriptive variable name? I had to read a few lines to figure out that `sa` is likely short for "service account".

Reply:
ok i can make that more descriptive.